.rubocop.yml

```diff
@@ -1,3 +1,8 @@
 ---
 AllCops:
-  TargetChefVersion: 17.0
```

Comment:
What was the reasoning for removing this line?

Reply:
it throws an error in cookstyle